lib/forking/forked_blockchain.js

```diff
@@ -800,4 +800,11 @@ ForkedBlockchain.prototype.getBlockLogs = function(number, callback) {
   });
 };
 
+ForkedBlockchain.prototype.close = function(callback) {
+  if (this.fork.disconnect) {
+    this.fork.disconnect();
+  }
+  BlockchainDouble.prototype.close.call(this, callback);
+};
```

Comment:
This function was necessary to get the websockets test to stop. without this mocha won't exit because of the open websocket connection